services/wallet/transfer/transaction.go

```diff
@@ -188,15 +188,21 @@ func rowsToMultiTransactions(rows *sql.Rows) ([]*MultiTransaction, error) {
 	return multiTransactions, nil
 }
 
+func getMultiTransactionTimestamp(multiTransaction *MultiTransaction) uint64 {
+	if multiTransaction.Timestamp != 0 {
+		return multiTransaction.Timestamp
+	}
+	return uint64(time.Now().Unix())
```

Comment:
is there an expected, valid situation where the Tx shouldn't have a timestamp and we should return Now() instead? If not, perhaps add a log here.